src/DssSpell.t.sol

```diff
@@ -19,6 +19,15 @@ interface SpellLike {
     function cast() external;
 }
 
+interface ChainLogLike {
```

Comment:
you don't need these here, you should have access to all of the interfaces in these test files via the import on line #6

Reply:
done